documentation.css

```diff
@@ -0,0 +1,173 @@
+*{
```

Comment:
I see you have used pixels everywhere.Use rem, em, vw, vh & % instead. Also, create variables for spacing(margins & padding), sizing, colors & fonts.

Ex - 

--space-s: 0.5rem;
--space-m: 1rem;
--space-l: 2rem;

This way, you can size your whole site easily. You would not even have to remember the values anymore.

Reply:
You need to do this for all the CSS files by the way